platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygTextView.swift

```diff
@@ -150,16 +150,12 @@ public class WysiwygTextView: UITextView {
     override public func caretRect(for position: UITextPosition) -> CGRect {
         // Compute system expected caret rect.
         let rect = super.caretRect(for: position)
-        // Determine rect for glyph at expected position.
-        let index = offset(from: beginningOfDocument, to: position)
-        let glyphRange = layoutManager.glyphRange(forCharacterRange: .init(location: index, length: 1), actualCharacterRange: nil)
-        let glyphRect = layoutManager.boundingRect(forGlyphRange: glyphRange, in: textContainer)
         // Use the system caret rect for `x` position and width and correct
-        // the `y` position and the height using the text glyphs.
+        // the `y` position and the height.
```

Comment:
Question: what if delete the full override `caretRect(for:)` at this point?
Made a quick try and LGTM.
wdyt?

Reply:
I think it's safe to keep it to increase the height of the caret.